src/sst/elements/ariel/arielcore.cc

```diff
@@ -218,6 +219,35 @@ void ArielCore::handleEvent(SimpleMem::Request* event) {
 	delete event;
 }
 
+void ArielCore::ISR_Opal(OpalEvent *ev) {
```

Comment:
@VamseeReddyK Can you pull the latest version of devel and merge it into your fork/branch? It has a new method for stalling ariel cores which you should be able to use instead of setting/clearing maxIssuePerCycle. See the 'isStalled' variable.

Reply:
@gvoskuilen I have some more commits in my branch for which I want to create a PR. Do I need to close this PR, merge disagg_amro branch with devel and then create a new PR?